renderer/components/Profile/ProfilePaneSignMessage/messages.js

```diff
@@ -0,0 +1,13 @@
+import { defineMessages } from 'react-intl'
+
+/* eslint-disable max-len */
+export default defineMessages({
+  sign_message_pane_title: 'Sign Message',
+  feature_desc: `This feature allows you to sign arbitrary messages to prove that you are in control of the node's public key. Signed message can be independently verified by the people you hand the message to.`,
```

Comment:
I'd suggest writing this in the imperative.

ie `Sign arbitrary messages to prove that you are in control of the node's public key....`